rmgpy/reactionTest.py

```diff
@@ -257,6 +257,18 @@ def test_methyl_adsorption_sticking_coefficient(self):
     def test_get_rate_coefficient_units_from_reaction_order(self):
 
         self.assertEqual(self.rxn1s.generate_reverse_rate_coefficient().A.units, 'm^2/(mol*s)')
+        
+    def test_equilibrium_constant_surface_kc(self):
+        """
+        Test the Reaction.get_equilibrium_constant() method.
+        """
+        Tlist = numpy.arange(400.0, 1600.0, 200.0, numpy.float64)
+        Kclist0 = [float(v) for v in
```

Comment:
why do this float(v) for v in list of strings, instead of just doing a list of floats?

Reply:
Not sure, I notice other tests were written this way, so I followed that